scripts/create_service.sh

```diff
@@ -14,6 +14,8 @@ port="$3"
 hostname_suffix="$4"
 shift 4 # Remaining arguments ($@) are passed to the server binary.
 
+repo_version=${repo_version-"master"}
```

Comment:
optional: Should we make it:

1. also able to be configured in `all.sh`, so lazy users (like me) can run one script and still benefit from this change;
2. a command line argument or an environmental variable for people, so people can update more easily.

Reply:
> 1. also able to be configured in all.sh, so lazy users (like me) can run one script and still benefit from this change;

`all.sh` runs `create_service.sh`, so it would transitively apply.

> 2. a command line argument or an environmental variable for people, so people can update more easily.

This is an environment variable. If `repo_version` is unset, "master" will be used.

Reply:
shell magic :mage:

Reply:
I meant an all cap environmental variable name, and talk about it in the comments or somewhere.

Reply:
🧙🦯✨

Reply:
I think this is not a well-defined convention either way.

The [IEEE POSIX spec](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap08.html) says:

> The name space of environment variable names containing lowercase letters is reserved for applications. Applications can define any environment variables with names from this name space without modifying the behavior of the standard utilities.

To me this sounds like a recommendation.

[Google's style guide](https://google.github.io/styleguide/shellguide.html#s7.3-constants-and-environment-variable-names) says:

> Constants and anything exported to the environment should be capitalized.

No justification is given.  I'm fine with changing it if you think it will help.

Reply:
I don't have a strong opinion. The merged version looks good to me 👍